lib/dpl/provider/atlas.rb

```diff
@@ -16,14 +16,16 @@ class Atlas < Provider
             chmod +x $HOME/bin/gimme
           fi
 
-          export GOPATH="$HOME/gopath:$GOPATH"
-          eval "$(gimme 1.4.2)" &>/dev/null
+          if [ -z $GOPATH ]; then
+            export GOPATH="$HOME/gopath"
+          else
+            export GOPATH="$HOME/gopath:$GOPATH"
+          fi
+          eval "$(gimme 1.6)" &> /dev/null
 
+          unset GIT_HTTP_USER_AGENT
```

Comment:
I honestly don't know, I didn't really look at how the shell handling works to determine if all deploy providers run within the same shell environment, but I could test it by adding another provider to my test repo